src/energy_comms/transform/epa.py

```diff
@@ -0,0 +1,29 @@
+import pandas as pd
+
+
+def transform(df):
+
```

Comment:
I think all of your column name transformations can be done at the beginning of this transform function so that the names used for the rest of the function are standardized. It's also nice to enforce types on important columns (zip code already read in as a string hopefully). If you really only want these five columns you can filter that here too:

```
df = df.astype({"site_name": "string", "state": "string", "latitude": "float", "longitude: "float"})
df.columns = df.columns.str.lower().str.replace(" ", "_")
df["site_name"] = df["site_name"].str.title()
df = df[["site_name", "zip_code", "latitude", "longitude", "state"]]
```